sdks/java/core/src/test/java/org/apache/beam/sdk/runners/PipelineRunnerTest.java

```diff
@@ -39,4 +49,36 @@ public void testInstantiation() {
     PipelineRunner<?> runner = PipelineRunner.fromOptions(options);
     assertTrue(runner instanceof CrashingRunner);
   }
+
+  private static class ScaleFn<T extends Number>
+      extends SimpleFunction<T, Double> {
+
+    private final double scalar;
+
+    public ScaleFn(double scalar) {
+      this.scalar = scalar;
+    }
+
+    @Override
+    public Double apply(T input) {
+      return scalar * input.doubleValue();
+    }
+  }
+
+  @Test
+  @Category(ValidatesRunner.class)
+  public void testRunPTransform() {
+    PipelineRunner pipelineRunner = PipelineRunner.create();
```

Comment:
Move this down to the end to make it clear the runner is out of the picture 'till the end. (Maybe you don't even need a temporary...)

Reply:
Actually, perhaps this could even be written inline. E.g.

```
PipelineRunner.create().run(
    new PTransform<PBegin, POutput>() {
        ...
    });
```

Reply:
Done.